medusa/name_parser/parser.py

```diff
@@ -300,10 +301,12 @@ def _parse_string(self, name):
         elif result.season_number and result.episode_numbers:
             new_episode_numbers, new_season_numbers, new_absolute_numbers = self._parse_series(result)
 
+        # Remove None from the list of seasons, as we can't sort on that
+        new_season_numbers = sorted(set(list(filter(lambda season: season is not None, new_season_numbers))))
```

Comment:
I think you can remove the first list():
`sorted(set(filter(lambda season: season is not None, new_season_numbers)))`

Reply:
Ooh yeah. Your right.